server/service/src/sync/test/integration/omsupply_central/test.rs

```diff
@@ -0,0 +1,11 @@
+#[cfg(test)]
+mod tests {
+    use crate::sync::test::integration::omsupply_central::{
+        pack_variant::PackVariantTester, test_omsupply_central_records,
+    };
+
+    #[actix_rt::test]
+    async fn omsupply_central_sync_pack_variant() {
```

Comment:
Can/should this be renamed to also start with `integration_sync` so that it will run with all the other integration test when doing: `cargo test integration_sync`?

Reply:
Thanks, great point: 95de51c2afcb49393a1a388eae0d365efeffef36